docs/models/ErrorCode.md

```diff
@@ -0,0 +1,10 @@
+# Fingerprint.ServerSdk.Model.ErrorCode
+Error code: * `request_cannot_be_parsed` - The query parameters or JSON payload contains some errors   that prevented us from parsing it (wrong type/surpassed limits). * `secret_api_key_required` - secret API key in header is missing or empty. * `secret_api_key_not_found` - No Fingerprint application found for specified secret API key. * `public_api_key_required` - public API key in header is missing or empty. * `public_api_key_not_found` - No Fingerprint application found for specified public API key. * `subscription_not_active` - Fingerprint application is not active. * `wrong_region` - Server and application region differ. * `feature_not_enabled` - This feature (for example, Delete API) is not enabled for your application. * `request_not_found` - The specified event ID was not found. It never existed, expired, or it has been deleted. * `visitor_not_found` - The specified visitor ID was not found. It never existed or it may have already been deleted. * `too_many_requests` - The limit on secret API key requests per second has been exceeded. * `state_not_ready` - The event specified with event ID is   not ready for updates yet. Try again.   This error happens in rare cases when update API is called immediately   after receiving the event ID on the client. In case you need to send   information right away, we recommend using the JS agent API instead. * `failed` - Internal server error. * `event_not_found` - The specified event ID was not found. It never existed, expired, or it has been deleted. * `missing_module` - The request is invalid because it is missing a required module. * `payload_too_large` - The request payload is too large and cannot be processed. 
```

Comment:
Let's fix formatting to keep line breaks (it was working in the previous version) and remove the useless empty table?

Reply:
Updated model documentation with commit [#f3bf159](https://github.com/fingerprintjs/fingerprint-pro-server-api-dotnet-sdk/pull/126/commits/f3bf159397eab2970c9183856631e52d272179e2)